src/vm/runners/builtin_runner/keccak.rs

```diff
@@ -679,7 +679,10 @@ mod tests {
             ((0, 35), 0)
         ];
 
-        let keccak_instance = KeccakInstanceDef::new(2048, vec![1; 8]);
+        let keccak_instance = KeccakInstanceDef {
+            _state_rep: vec![1; 8],
+            ..Default::default()
+        };
```

Comment:
I'm not too sure why this change is needed.

Reply:
it was a clippy error.  It had been fixed on main, since, he will rebase and it won't appear anymore